drivers/sdhc/rcar_mmc.c

```diff
@@ -827,14 +827,18 @@ static int rcar_mmc_sd_buf_rx_tx_data(const struct device *dev,
 		return -EINVAL;
 	}
 
+	start_data_xref_us = k_ticks_to_us_ceil32(k_uptime_ticks());
 	for (block = 0; block < data->blocks; block++) {
 		uint8_t *buf = (uint8_t *)data->data + (block * data->block_size);
 		uint32_t info2_reg;
 		uint16_t w_off; /* word offset in a block */
+		uint32_t elapsed_us = k_ticks_to_us_ceil32(k_uptime_ticks()) - start_data_xref_us;
```

Comment:
4294967296 microseconds is only 4295 seconds, which is 1.2 hours. This means that overflow event is very likely around there.